bugbear.py

```diff
@@ -1510,6 +1521,16 @@ def visit_Lambda(self, node):
         " decorator. Consider adding @abstractmethod."
     )
 )
+B028 = Error(
+    message=(
+        "B028 No explicit stacklevel keyword argument found. The warn method from the"
+        " warnings module uses a stacklevel of 1 by default. This will only show a"
+        " stack trace for the line on which the warn method is called."
+        " It is therefore recommended to use a stacklevel of 2 or"
+        " greater to provide more information to the user. This warning can be disabled"
+        " by explicitly setting a stacklevel."
```

Comment:
```suggestion
        " greater to provide more information to the user."
```
I feel the opening sentence states this.

Reply:
Thanks! Updated now.